ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Overview/01_General.md

# General information

> The requirements for modern accounting software go far beyond document posting. The *Accounting* module includes all functions of accounts receivable and accounts payable and can meet even the most complex of requirements.

Reviewer: ```suggestion
The requirements for modern accounting software go far beyond document posting. The *Accounting* module includes all functions of accounts receivable and accounts payable and can meet even the most complex  requirements.
```

> The requirements for modern accounting software go far beyond document posting. The *Accounting* module includes all functions of accounts receivable and accounts payable and can meet even the most complex of requirements.

Reviewer: ```suggestion
The requirements for modern accounting software go far beyond document posting. The *Accounting* module includes all functions of accounts receivable and accounts payable and can meet even the most complex requirements.
```

> The *Actindo Core1 Platform* is the only middleware worldwide with a complete accounting module integrated. This means that all bookings for all orders of the connected channels and accounts are automatically generated and optionally transferred live to a group accounting system. As a result, your data integration is extremely simplified and easy to manage.

Reviewer: ```suggestion
The *Actindo Core1 Platform* is the only digital operations platform worldwide with a complete accounting module integrated. This means that all bookings for all orders of the connected channels and accounts are automatically generated and optionally transferred live to a group accounting system. As a result, your data integration is extremely simplified and easy to manage.
```
(Ich glaube, wir sollen das neue Wording verwenden)

> The Actindo Core1 platform is the only middleware worldwide with a complete accounting module integrated. This means that all bookings for all orders of the connected channels and accounts are automatically generated and optionally transferred live to a group accounting system. As a result, your data integration is extremely simplified and easy to manage.

Reviewer 2: ```suggestion
The *Actindo Core1 Platform* is the only middleware worldwide with a complete accounting module integrated. This means that all bookings for all orders of the connected channels and accounts are automatically generated and optionally transferred live to a group accounting system. As a result, your data integration is extremely simplified and easy to manage.
```

This documentation refers to *Accounting* module version 4.1.1. There may be differences in the UI, features and operation if your *Accounting* module is installed in a different version.

## Benefits

The *Accounting* module offers the following benefits:

> - Comprehensive business and tax evaluations, reports and analyses directly in the *Actindo Core1 Platform*  

Author: ```suggestion
- comprehensive business and tax evaluations, reports and analyses directly in the *Actindo Core1 Platform*  
```

> - Comprehensive business and tax evaluations, reports and analyses directly in the Actindo Core1 platform  

Reviewer 2: ```suggestion
- Comprehensive business and tax evaluations, reports and analyses directly in the *Actindo Core1 Platform*  
```
